libs/spark/src/Unstable_SectionMessage/Unstable_SectionMessage.tsx

```diff
@@ -0,0 +1,211 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import Paper, { PaperProps } from '@material-ui/core/Paper';
+import {
+  Unstable_AlertOctagon,
+  Unstable_AlertTriangle,
+  Unstable_CheckCircle2,
+  Unstable_Cross,
+  Unstable_Info,
+} from '../internal';
+import makeStyles from '../makeStyles';
+import { StandardProps } from '../utils';
+
+interface Unstable_SectionMessageTitleProps
+  extends StandardProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    Unstable_SectionMessageTitleClassKey
+  > {
+  /**
+   * The content of the component.
+   */
+  children?: string;
+}
+
+type Unstable_SectionMessageTitleClassKey = 'root';
+
+const useStylesTitle = makeStyles<Unstable_SectionMessageTitleClassKey>(
+  (theme) => ({
+    root: {
+      ...theme.unstable_typography.label,
+      color: theme.unstable_palette.text.heading,
+      marginBottom: 4,
+    },
+  }),
+  { name: 'MuiSparkUnstable_SectionMessageTitle' }
+);
+
+const Unstable_SectionMessageTitle = function Unstable_SectionMessageTitle(
+  props: Unstable_SectionMessageTitleProps
+) {
+  const { children, className } = props;
+
+  const classes = useStylesTitle();
+
+  return <div className={clsx(classes.root, className)}>{children}</div>;
+};
```

Comment:
```suggestion
```
Sorry I suppose I was unclear. I didn't mean to move the previous file contents as-is into `Unstable_SectionMessage`. I rather meant to combine them into one component.